src/tests/Common/helixpublishwitharcade.proj

```diff
@@ -118,6 +118,7 @@
 
   <PropertyGroup Condition="'$(TargetOS)' == 'Android' or '$(TargetOS)' == 'iOS' or '$(TargetOS)' == 'iOSSimulator'">
     <IncludeXHarnessCli>true</IncludeXHarnessCli>
+    <MicrosoftDotNetXHarnessCLIVersion>1.0.0-prerelease.20322.1</MicrosoftDotNetXHarnessCLIVersion>
```

Comment:
There should be a property available from `Versions.props` named `($MicrosoftDotNetXHarnessCLIVersion)`

Reply:
Remove.